content/communities/multilingual.md

```diff
@@ -5,7 +5,7 @@
 slug: multilingual
 date: 2014-02-21 4:58:10 -0400
 title: "Multilingual "
```

Comment:
Is the extra space in title necessary?

Reply:
Good catch, I have removed the extra white space